src/icons/fileIcons.ts

```diff
@@ -471,6 +471,7 @@ export const fileIcons: FileIcons = {
         { name: 'haml', fileExtensions: ['haml'] },
         { name: 'yang', fileExtensions: ['yang'] },
         { name: 'mjml', fileExtensions: ['mjml'] },
+        { name: 'now', fileExtensions: ['now.json'], light: true },
```

Comment:
Could you please use the `fileNames` attribute here instead of `fileExtensions`. "now.json" is not a file extension but a whole file name.

Reply:
Thanks for the comment!
It is fixed now.